composer.json

```diff
@@ -11,18 +11,18 @@
         { "name": "Knplabs", "homepage": "http://knplabs.com" }
     ],
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.1",
         "doctrine/common": "^3.3",
         "doctrine/persistence": "^2.5|^3.0",
         "doctrine/dbal": "^3.3",
         "doctrine/orm": "^2.12",
-        "doctrine/doctrine-bundle": "^2.6",
-        "symfony/cache": "^5.4|^6.0",
-        "symfony/dependency-injection": "^5.4|^6.0",
-        "symfony/http-kernel": "^5.4|^6.0",
-        "symfony/security-core": "^5.4|^6.0",
-        "symfony/framework-bundle": "^5.4|^6.0",
-        "symfony/string": "^5.4|^6.0",
+        "doctrine/doctrine-bundle": "^2.7.2",
```

Comment:
Does this constraint need to be updated?

Reply:
The #AsDoctrineListener attribute was introduced in DoctrineBundle 2.7.2